alerts.php

```diff
@@ -8,6 +8,12 @@
 
 $templatelist = 'myalerts_alert_row_popup,myalerts_alert_row_popup_no_alerts,myalerts_modal_content';
 
+define('THIS_SCRIPT', 'alerts.php');
```

Comment:
This is a duplicate of Line 7:

https://github.com/MyBBStuff/MyAlerts/pull/251/files#diff-974cd197043dd572368c0a2c492f22bcR7

Should be removed.